goosebit/plugins/__init__.py

```diff
@@ -0,0 +1,10 @@
+from importlib.metadata import entry_points
+
+from goosebit.settings import PLUGINS
+
+entries = entry_points(group="goosebit.plugins")
+
+plugins = []
+
+for p in PLUGINS:
+    plugins.extend(entries.select(name=p))
```

Comment:
Maybe iterate plugins from configuration file by key, passing the plugin values as settings to plugin class `__init__`.

I think you can probably use something [import_module](https://docs.python.org/3/library/importlib.html#importlib.import_module) based like [these functions](https://github.com/django/django/blob/stable/5.1.x/django/utils/module_loading.py#L8-L35) for module loading from settings defined string keys.

Reply:
When PR lol